ci/ci/github.py

```diff
@@ -318,6 +331,23 @@ async def post_github_status(self, gh_client, gh_status):
         except aiohttp.client_exceptions.ClientResponseError:
             log.exception(f'{self.short_str()}: Unexpected exception in post to github: {data}')
 
+    async def assign_gh_reviewer_if_requested(self, gh_client):
+        if len(self.assignees) == 0 and len(self.reviewers) == 0:
+            assignees = set()
+            if ASSIGN_SERVICES in self.body:
+                assignees.add(select_random_teammate(SERVICES_TEAM).gh_username)
+            if ASSIGN_COMPILER in self.body:
+                assignees.add(select_random_teammate(COMPILER_TEAM).gh_username)
+            data = {'assignees': list(assignees)}
+            try:
+                await gh_client.post(
+                    f'/repos/{self.target_branch.branch.repo.short_str()}/issues/{self.number}/assignees', data=data
+                )
+            except gidgethub.HTTPException:
+                log.info(f'{self.short_str()}: post assignees to github failed due to exception: {data}', exc_info=True)
```

Comment:
this is really an error log, right?

Reply:
Ya there were actually a bunch of this littered around github.py. I updated the exceptional logs to `exception`.